documentation/flreconstruct/FLReconstruct.md

```diff
@@ -335,20 +335,20 @@ $ flsimulate -c simu.conf -o example.brio
 ...
 ~~~~~
 
-Running the visualization, we can display the first event where we can
+The resulting `example.brio` file can be opened with the `flvisualize`
+application. We can visualize the first event where we can
 see plenty of raw truth Geiger hits and one single calorimeter hit, as
 well as the  position of the decay vertex (purple  cross on the source
 foil):
```

Comment:
Presumably if you ran it yourself you might not see 1 single calorimeter hit? Or does simu.conf  set all the random seeds so that you will get exactly the same output every time?
I think the picture here is using the old colour scheme, obviously that is not a big deal, but maybe worth noting.